tools/create_certs/generate_CA_backup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Do you still need anything from this backup file?  If not, let's remove it so future us don't get confused :)

Reply:
deleted!